Makefile

```diff
@@ -473,24 +443,41 @@ push-to-manifest-repo:
 	sed -i -e 's,ICON_BASE64_DATA,$(shell base64 --wrap=0 ./assets/icon/red-hat-logo.svg),g' $(MANIFESTS_TMP)/${BUNDLE_VERSION}/*.clusterserviceversion.yaml
 	sed -i -e 's,ICON_MEDIA_TYPE,image/svg+xml,g' $(MANIFESTS_TMP)/${BUNDLE_VERSION}/*.clusterserviceversion.yaml
 
-.PHONY: prepare-bundle-to-quay
-## Prepare manifest bundle to quay application
-prepare-bundle-to-quay:
+.PHONY: verify-operator-manifests
+## verify operator manifests
+verify-operator-manifests: prepare-operator-manifests
 	$(Q)python3 -m venv $(PYTHON_VENV_DIR)
 	$(Q)$(PYTHON_VENV_DIR)/bin/pip install --upgrade setuptools
 	$(Q)$(PYTHON_VENV_DIR)/bin/pip install --upgrade pip
 	$(Q)$(PYTHON_VENV_DIR)/bin/pip install operator-courier==2.1.2
 	$(Q)$(PYTHON_VENV_DIR)/bin/operator-courier --version
-	$(Q)$(PYTHON_VENV_DIR)/bin/operator-courier verify $(MANIFESTS_TMP)
+	$(Q)$(PYTHON_VENV_DIR)/bin/operator-courier $(VERBOSE_FLAG) verify $(MANIFESTS_TMP)
 	rm -rf deploy/olm-catalog/$(GO_PACKAGE_REPO_NAME)/$(BUNDLE_VERSION)
 
-
-.PHONY: push-bundle-to-quay
-## Push manifest bundle to quay application
-push-bundle-to-quay:
-	$(Q)$(PYTHON_VENV_DIR)/bin/operator-courier verify $(SBR_MANIFESTS)
-	$(Q)$(PYTHON_VENV_DIR)/bin/operator-courier push $(SBR_MANIFESTS) redhat-developer service-binding-operator $(BUNDLE_VERSION) "$(QUAY_BUNDLE_TOKEN)"
-
+.PHONY: build-bundle-image
+## build bundle image
+build-bundle-image: verify-operator-manifests
+	$(Q)operator-sdk bundle create --directory $(MANIFESTS_TMP)/$(BUNDLE_VERSION) -b $(CONTAINER_RUNTIME) --package $(CSV_PACKAGE_NAME) --channels beta,alpha --default-channel beta $(VERBOSE_FLAG) $(OPERATOR_BUNDLE_IMAGE_REF)
+
+.PHONY: push-bundle-image
+## push bundle image
+push-bundle-image: build-bundle-image
+	$(Q)$(CONTAINER_RUNTIME) push $(OPERATOR_BUNDLE_IMAGE_REF)
+	$(Q)operator-sdk bundle validate -b $(CONTAINER_RUNTIME) $(OPERATOR_BUNDLE_IMAGE_REF)
+
+.PHONY: build-index-image
+## build index image
+build-index-image:
+	$(Q)opm index add -u $(CONTAINER_RUNTIME) -p $(CONTAINER_RUNTIME) --bundles $(OPERATOR_BUNDLE_IMAGE_REF) --tag $(OPERATOR_INDEX_IMAGE_REF)
+
+.PHONY: push-index-image
+## push index image
+push-index-image: build-index-image
+	$(Q)$(CONTAINER_RUNTIME) push $(OPERATOR_INDEX_IMAGE_REF)
+
+.PHONY: release-operator-bundle
+## Build and release operator bundle and operator bundle index
+release-operator-bundle: push-operator-image push-bundle-image push-index-image
```

Comment:
we here release the operator, not just bundle, so how about to name this target as `reelase-operator`?